test/src/test/java/org/corfudb/infrastructure/AutoCommitServiceTest.java

```diff
@@ -0,0 +1,146 @@
+package org.corfudb.infrastructure;
+
+import org.corfudb.protocols.wireprotocol.DataType;
+import org.corfudb.protocols.wireprotocol.LogData;
+import org.corfudb.protocols.wireprotocol.TokenResponse;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.view.AbstractViewTest;
+import org.corfudb.runtime.view.Layout;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Created by WenbinZhu on 9/30/19.
+ */
+public class AutoCommitServiceTest extends AbstractViewTest {
```

Comment:
Can we have some test that exercises changing the primary sequencer right in the middle of auto commit? ... so two auto commits run in parallel.

Reply:
Done. Added a new test.